app/src/main/java/com/muharuto/introduceclubactivities/add/AddClubSummaryFragment.kt

```diff
@@ -2,21 +2,105 @@ package com.muharuto.introduceclubactivities.add
 
 import android.os.Bundle
 import android.view.View
+import android.widget.TextView
 import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import com.muharuto.introduceclubactivities.ClubSummaryApplication
 import com.muharuto.introduceclubactivities.R
+import com.muharuto.introduceclubactivities.database.clubsummarydata.ClubSummaryData
 import com.muharuto.introduceclubactivities.databinding.FragementAddClubSummaryBinding
+import com.muharuto.introduceclubactivities.detail.ClubViewModel
+import com.muharuto.introduceclubactivities.detail.ClubViewModelFactory
 
 class AddClubSummaryFragment : Fragment(R.layout.fragement_add_club_summary) {
-    private var fragmentAddClubSummaryBinding: FragementAddClubSummaryBinding? = null
+
+    private val navigationArgs: AddClubSummaryFragmentArgs by navArgs()
+    lateinit var clubSummaryData: ClubSummaryData
+
+
+    private var _fragmentAddClubSummaryBinding: FragementAddClubSummaryBinding? = null
+    private val fragmentAddClubSummaryBinding get() = _fragmentAddClubSummaryBinding!!
```

Comment:
```suggestion
    private var fragmentAddClubSummaryBinding: FragementAddClubSummaryBinding? = null
```
こちらのような書き方の方がシンプルな気がするのですがいかがでしょう？

Reply:
恐らくコードラボの書き方を参考にしてくれたと思うんだけど、以前の書き方の方が良いかなと自分も思います～！